packages/mui-material/src/BottomNavigationAction/BottomNavigationAction.js

```diff
@@ -116,20 +119,44 @@ const BottomNavigationAction = React.forwardRef(function BottomNavigationAction(
     }
   };
 
+  const externalForwardedProps = {
+    slots,
+    slotProps,
+  };
+
+  const [RootSlot, rootProps] = useSlot('root', {
+    elementType: BottomNavigationActionRoot,
```

Comment:
```suggestion
    shouldForwardComponentProp: true,
    elementType: BottomNavigationActionRoot,
```

The `BottomNavigationActionRoot` comes from a `ButtonBase`.